pynance/textimporter.py

```diff
@@ -0,0 +1,205 @@
+import csv
+import datetime
+
+import pandas as pd
+import numpy as np
+
+
+class CsvImporter():
+    """
+    Importer class to read csv files to well defined pandas.DataFrames
+    """
+    def __init__(self):
+        pass
+
+    def read(self, filepath_or_buffer, cfd):
```

Comment:
For the sake of speaking names, I propose to replace `cfd` by something like `description`.